source/templates.tex

```diff
@@ -583,18 +583,17 @@
   id-expression
 \end{bnf}
 
-\begin{note}
```

Comment:
Is this change relevant to the subject at hand?

Reply:
I think so. The below "name must be known to refer to a template" is supposed to be what normatively states what is and is not a *template-name*, but is hopelessly vague, and changing the "must" to a "shall" is also wrong -- the right thing is to say that an *identifier* is a *template-name* if name lookup associates it with one, which is what the note prior to the broken sentence says.

That said, I'd be happy to take a less-invasive alternate path through here.

Reply:
Let's just leave this one alone for C++17. I'm now concerned that this sentence may be intended to have normative impact (eg, what else would make `sizeof &operator""_foo < 8 > 0` parse as a comparison rather than as having a *template-argument-list*?)

Reply:
Done.